### PR TITLE
wgpu: Update wgpu and naga to 0.19

### PR DESCRIPTION
**Pull request by Dinnerbone**
0.19 is out!

This depends on:
- [x] #14703  (due to egui already having updated winit)
- [x] https://github.com/emilk/egui/pull/3824
- [x] https://github.com/bevyengine/naga_oil/pull/73

### CODE REVIEW

File: tests/tests/swfs/avm1/netstream_seek_flv/output.expected.png
Comment (torokati44) [inline tests/tests/swfs/avm1/netstream_seek_flv/output.expected.png:1]: Well this does not look ideal... :sweat_smile:

Comment (Dinnerbone) [reply tests/tests/swfs/avm1/netstream_seek_flv/output.expected.png:1]: Removed it from this PR but it's still missing from master :D

**Comment by Dinnerbone** [comment]
I've marked this ready as review, but it depends on a commit of egui repo as it's not yet released.

**Review by Aaron1011 — APPROVED** [review]

**Review by sleepycatcoding — APPROVED** [review]
LGTM

**Pull request merged**